simplq/src/store/appSlice.js

```diff
@@ -19,6 +20,26 @@ const appSlice = createSlice({
       state.notificationPermission = action.payload;
     },
   },
+  extraReducers: {
+    [fetchQueues.pending]: (state) => {
+      state.infoText = `Loading queues...`;
+    },
+    [fetchQueues.rejected]: (state, action) => {
+      state.errorText = action.error.message;
+    },
+    [fetchQueues.fulfilled]: (state, action) => {
+      state.infoText = `Number of queues fetched: ${action.payload.queues.length}`;
+    },
+    [deleteQueue.pending]: (state) => {
+      state.infoText = `Deleting queue...`;
+    },
+    [deleteQueue.rejected]: (state, action) => {
+      state.errorText = action.error.message;
+    },
+    [deleteQueue.fulfilled]: (state, action) => {
+      state.infoText = `Deleted ${action.payload.queueName}`;
+    },
+  },
```

Comment:
I'll understand and study this later. It should ideally render spinners at the right places. But can we can handle this later?

Reply:
That's the real beauty of this. We can:
- match all pending requests and have only one state with one or many spinners looking at it;
- have separate loading state and spinner for every async action;
- make one async action wait for another, that waits for another... and update state when it's all over;
- DO TI LATER 😄